src/platform/forms-system/src/js/fields/ArrayField.jsx

```diff
@@ -389,10 +391,8 @@ export default class ArrayField extends React.Component {
                   <Element name={`table_${itemIdPrefix}`} />
                   <div className="row small-collapse">
                     <div className="small-12 columns va-growable-expanded">
-                      {isLast && items.length > 1 ? (
-                        <h3 className="vads-u-font-size--h5">
-                          New {uiOptions.itemName || 'item'}
-                        </h3>
+                      {isLast && multipleRows ? (
+                        <h3 className="vads-u-font-size--h5">New {itemName}</h3>
```

Comment:
No longer need the `|| 'item'` logic because we are reusing the `itemName` variable that already has that in its instantiation

Reply:
I saw this when I was working in here too and got pretty confused. If you follow the trail of commits, it used to use that `itemName` variable, but was later removed. I think that's because `itemName` could theoretically be that aria label function thing which is different than the supplied `uiOptions.itemName`. I think someone got confused in the past and property/variable names got switched up or something. I really don't know. 

Here's Robin adding the `itemName` variable: https://github.com/department-of-veterans-affairs/vets-website/commit/1538e8b16395135c9e97ca7949b9616ed650d7b6#diff-77e9e96767e66321e66766ec6d502a8fa9c8f3f83a3e6fb2e10c9b7bcad66d66R292

And here's obliviga going back to `uiOptions.itemName`, but not adding back the `|| item` piece: https://github.com/department-of-veterans-affairs/vets-website/commit/316e37f9ff585e73325de39aa9d3e7ea00368d7e#diff-77e9e96767e66321e66766ec6d502a8fa9c8f3f83a3e6fb2e10c9b7bcad66d66R287

Reply:
And then I obviously added the `|| item` part back: https://github.com/department-of-veterans-affairs/vets-website/commit/766227d8e2899c40663890674ae196d98b447f1d#diff-77e9e96767e66321e66766ec6d502a8fa9c8f3f83a3e6fb2e10c9b7bcad66d66R394

Reply:
In conclusion, this file is a mess and there have been so many `uiOptions` added over the years that I really have no idea which ones are used where.

And to make things more fun, you've got this file: https://github.com/department-of-veterans-affairs/vets-website/blob/main/src/platform/forms-system/src/js/review/ArrayField.jsx

Which is supposed to the array field that gets rendered on the review page, but it gets **completely bypassed** if you use `keepInPageOnReview`, which 4142 and almost every other form has used to make the array work on the review page.